### ISSUE TITLE
doubts regarding Bert Innerworking

### ISSUE THREAD

**Issue by apkbala107** [gmihaila/ml_things#23]
hi,

          i read your article bert inner working.. it is very informative. i have some questions regarding this article. that is, how to show the final sentiment of the two statements?

'I love cats!',
              "He hates pineapple pizza."

**Comment by gmihaila** [comment]
you check the logits of the two examples.
